hutch_python/load_conf.py

```diff
@@ -456,11 +468,19 @@ def load_conf(conf, hutch_dir=None, args=None):
 
     # Happi db and Lightpath
     if db is not None:
+
+        # Get the names of all devices that should not be loaded and
+        # pass it to get_happi_objs()
+        if exclude_devices:
+            exclude_devices = [
+                device_name.strip() for device_name in exclude_devices.split(',')]
```

Comment:
For more context, YAML config files support lists of strings natively, so maybe we can expect the user to use these instead of making a string that need to parse.

Multi-line array of strings:
```
key:
  - value1
  - value2
  - value3
```
Alternatively in one line:
```
key: ['value1', 'value2', 'value3']
```
I think this PR currently assumes we instead have this encoded in a string as:
```
key: "value1, value2, value3"
```

Reply:
That's a nice feature of YAML. I updated load_conf.py to assume `exclude_devices` in conf.yml will have a multi-line array of strings or an actual list.